internal/sleuth/code_change_source_resource.go

```diff
@@ -218,9 +218,9 @@ func (ccsr *codeChangeSourceResource) Schema(_ context.Context, _ resource.Schem
 			"project_slug": schema.StringAttribute{
 				MarkdownDescription: "The slug of the project that this code change source belongs to.",
 				Required:            true,
-				PlanModifiers: []planmodifier.String{
-					stringplanmodifier.RequiresReplace(), // ForceNew replacement
-				},
+				// PlanModifiers: []planmodifier.String{
+				// 	stringplanmodifier.RequiresReplace(), // ForceNew replacement
+				// },
```

Comment:
So this is the fix? Don't do a replace for a changed project? My guess is it was there because if you tried to change the project slug, we don't support moving a code source to a new project.

Reply:
Actually, thanks for questioning this. It definitely stopped it from recreating the resource, but it just dawned on me to figure out why the project slug was causing this even though it wasn't changing.

Turns out, terraform was just always setting the value as "(known after apply)" which meant it was always different.

The actual fix was to tell terraform to use the previous value in the project resource if there wasn't a new value explicitly passed in.

I made the change to this PR and now when I run it, I only get the update in place for the one project param I actually changed:

![image](https://github.com/user-attachments/assets/f8085b90-db9d-43d1-99f2-b5daf2fd4242)